functests/utils/consts.go

```diff
@@ -25,6 +32,18 @@ func init() {
 	}
 
 	NodesSelector = os.Getenv("NODES_SELECTOR")
+
+	NodeSelectorLabels = map[string]string{
```

Comment:
Suggestion, feel free to ignore (but it's one of my fixations :p) 
How about exposing this in the nodes package and rename it as `SelectorLabels`?
That would read as `nodes.SelectorLabels` when you call it.
The package name is part of the object name and we should use it. By relying on "utils" we gave up part of our naming structure.